src/Features/LanguageServer/Protocol/Features/UnifiedSuggestions/UnifiedSuggestedActionsSource.cs

```diff
@@ -207,9 +213,26 @@ static CodeFixGroupKey GetGroupKey(CodeFix fix)
                 return null;
             }
 
+            var document = fixAllState.Document!;
             using var fixAllSuggestedActionsDisposer = ArrayBuilder<IUnifiedSuggestedAction>.GetInstance(out var fixAllSuggestedActions);
             foreach (var scope in supportedScopes)
             {
+                if (scope is FixAllScope.ContainingMember or FixAllScope.ContainingType)
+                {
+                    // Skip showing ContainingMember and ContainingType FixAll scopes if the language
+                    // does not implement 'IFixAllSpanMappingService' langauge service or
+                    // we have no mapped FixAll spans to fix.
```

Comment:
this is a bit odd right?  other languages aren't going to add those FixAllScopes... so how would we get here?  we'd only get these scopes for C#/VB fixers taht update to use this, which means tehre is a mapping service...

Reply:
This is a defensive check in case a third-party code fix provider for a different language returns the new FixAllScopes for `FixAllProvider.GetSupportedFixAllScopes`, but the language itself does not support/implement these scopes.